### PR TITLE
remove indirect in depency graph

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: service/controller/v25/cloudformation/cloud_formation.go [hunk 1]
```diff
@@ -29,11 +29,11 @@ func New(config Config) (*CloudFormation, error) {
 // DescribeOutputsAndStatus returns stack outputs, stack status and error. The
 // stack status is returned when the error is nil or the error is matched by
 // IsOutputsNotAccessible.
-func (c *CloudFormation) DescribeOutputsAndStatus(stackName string) ([]*cloudformation.Output, string, error) {
+func (c *CloudFormation) DescribeOutputsAndStatus(stackName string) ([]Output, string, error) {
```
Comment (xh3b4sd) [inline service/controller/v25/cloudformation/cloud_formation.go]: We abstract certain external functionality but were still relying on the external types. This caused an asymmetry in the dependency graph and caused import weirdness where the abstraction got used. The inspiration here came from the ebs service implementation.

File: service/controller/v25/cloudformation/cloud_formation.go [hunk 2]
```diff
@@ -47,11 +47,11 @@ func (c *CloudFormation) DescribeOutputsAndStatus(stackName string) ([]*cloudfor
 			return nil, "", microerror.Mask(err)
 		}
 
-		if len(o.Stacks) > 1 {
+		if len(o.Stacks) != 1 {
```
Comment (xh3b4sd) [inline service/controller/v25/cloudformation/cloud_formation.go]: When there is ever no stack, the code below would panic.

File: service/controller/v25/resource/tccpoutputs/create.go [hunk 3]
```diff
@@ -5,11 +5,10 @@ import (
 	"fmt"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/cloudformation"
```
Comment (xh3b4sd) [inline service/controller/v25/resource/tccpoutputs/create.go]: This is what I was aiming for. Getting rid of two `cloudformation` packages that are imported in the same file.

**Comment by kopiczko** [comment]
I also like that <3

**Pull request merged**